apps/s_server.c

```diff
@@ -3544,6 +3544,7 @@ static int generate_session_id(SSL *ssl, unsigned char *id,
                                unsigned int *id_len)
 {
     unsigned int count = 0;
+    unsigned int session_id_prefix_len = strlen(session_id_prefix);
```

Comment:
A coding style nit: there should be a blank line after this line. (between variables declaration area and other code)

Reply:
Done. Thank you by review work.